packages/insomnia/src/ui/components/websockets/action-bar.tsx

```diff
@@ -79,17 +80,20 @@ export const WebSocketActionBar: FC<ActionBarProps> = ({ request, workspaceId, e
     }
     try {
       const renderContext = await getRenderContext({ request, environmentId, purpose: RENDER_PURPOSE_SEND });
-      const { url, headers, authentication } = request;
+      const { url, headers, authentication, parameters } = request;
       // Render any nunjucks tags in the url/headers/authentication settings
       const rendered = await render({
         url,
         headers,
         authentication,
+        parameters,
       }, renderContext);
+      const qs = buildQueryStringFromParams(rendered.parameters);
+      const urlWithQueries = joinUrlAndQueryString(rendered.url, qs);
       window.main.webSocket.create({
         requestId: request._id,
         workspaceId,
-        url: rendered.url,
+        url: urlWithQueries,
```

Comment:
Nitpick (if you accept the above suggestions):
```suggestion
        url,
```